src/com/mincom/ellipse/javatraining/springmvc/controller/ApplicationController.java

```diff
@@ -15,6 +15,7 @@
 public class ApplicationController {
  
 	private static final String INDEX_PAGE_KEY = "index";
+	private static final String ERROR_PAGE_KEY = "errorPage";
 	private static final String FINAL_PAGE_KEY = "finalPage";
 	private final String MESSAGE_MAP_KEY = "input_message";
 	private final static Logger logger = LoggerFactory.getLogger(ApplicationController.class);
```

Comment:
`private static final` is preferred, for `private static final` constant, the constant name should be all capitalized